packages/toolpad-app/prisma/schema.prisma

```diff
@@ -96,3 +95,13 @@ enum DomNodeAttributeType {
   jsExpressionAction
   navigationAction
 }
+
+model Connection {
+  id         String   @id @default(cuid())
+  name       String
+  createdAt  DateTime @default(now())
+  editedAt   DateTime @default(now())
+  datasource String
```

Comment:
Would it be worth defining enums in the database for fields like `datasource`?

Reply:
Not if we ever plan to allow users to define datasources as plugins.

Same as for `AppTemplateId`, this is not a good candidate for an enum since it's data and subject to receive dynamic or user defined values at some point.

Reply:
yeah that's a good point i hadn't considered. makes sense